app/src/main/java/org/dhis2/utils/customviews/CoordinatesView.java

```diff
@@ -306,6 +306,15 @@ public void onFocusChange(View v, boolean hasFocus) {
     @SuppressLint("MissingPermission")
     public void updateLocation(Geometry geometry) {
 
+        setCoordinatesValue(geometry);
+
+        this.currentGeometry = geometry;
+        if (listener2 != null)
+            listener2.onCurrentLocationClick(geometry);
```

Comment:
I know it was here, but if you can change the name listener2 please

Reply:
Yes, good idea